thunder/executors/nvfuserex_impl.py

```diff
@@ -441,7 +441,9 @@ class FusionDefinitionWrapper:
     cache_clear: None | Callable = None
     last_used: None | FusionDefinition = None
     last_inputs: None | Sequence[tuple] = None
-    store_inputs: bool = False
+    store_inputs: bool = (False,)
+    _enable_options: list = ([],)
+    _disable_options: list = ([],)
```

Comment:
Could you improve either the type hint or the default values?
Because the type annotations seem to contradict with the default values. The default ones are `tuple[bool]`, `tuple[list[Any]]`, and `tuple[list[Any]]`, respectively.